plans/no-rpm.fmf

```diff
@@ -0,0 +1,28 @@
+execute:
+    how: tmt
+prepare:
+    - name: packages
+      how: install
+      package: [ golang, podman-docker ]
+    - name: enable-linger
+      how: shell
+      script: loginctl enable-linger $(whoami)
+    - name: golangci
+      how: shell
+      script: |
+        # https://golangci-lint.run/welcome/install/
+        curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/HEAD/install.sh | sh -s -- -b /bin v2.1.6
+
+/validate:
+    execute+:
+        script: |
+            make validate
+            make validate-docs
+
+/integration:
+    execute+:
+        script: make test-integration
```

Comment:
This would try to run everything in a container (and it’s a barely-maintained test path not otherwise exercised in CI, AFAIK). Is this extra layer Intentional? It might not be necessary if everything were running in a dedicated VM. (I have ~no opinion.)